docs/source/en/training/text2image.mdx

```diff
@@ -155,6 +155,28 @@ python train_text_to_image_flax.py \
 </jax>
 </frameworkcontent>
 
+## Training with better convergence
+
+We support training with the Min-SNR weighting strategy proposed in [Efficient Diffusion Training via Min-SNR Weighting Strategy](https://arxiv.org/abs/2303.09556) which helps to achieve faster convergence
+by rebalancing the loss. In order to use it, one needs to set the `--snr_gamma` argument. The recommended
+value when using it is 5.0. 
```

Comment:
Out of curiosity, where is this value proposed? Is there a rule of thumb when choosing a value for this?

Reply:
It's reported in the paper. A gamma of 5.0 always leads to better results in the experiments presented by the authors in the paper.